src/Components/WebAssembly/BlazorManifest/src/WorkloadManifest.json.in

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "workloads": {
+    "microsoft-net-sdk-blazorwebassembly-aot": {
```

Comment:
this is user-facing name, it should be something like `microsoft-blazor-webassembly`